File: web-api/src/main/java/com/client/transaction/webapi/persistance/model/TransactionModel.java
package com.client.transaction.webapi.persistance.model;

import lombok.Getter;
import lombok.Setter;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;
import org.springframework.stereotype.Repository;

@Getter
@Setter
@Repository
@Document(collection = "transaction")
public class TransactionModel {

    @Id
    public int clientId;
    public double transactionsAmount;

    public TransactionModel() {
    }

    public TransactionModel(int clientId, double transactionsAmount) {
        this.clientId = clientId;
        this.transactionsAmount = transactionsAmount;
    }
}

File: web-app/src/app/models/transaction.model.ts

export class Transaction {

    _id: string;
    _clientId: string;
    transactionsAmount: number;
    transactionDate: any;

    constructor() {
        this._id = null;
        this._clientId = null;
        this.transactionsAmount = null
        this.transactionDate = null;
    }
}